### PR TITLE
fix: Content page side menu doesn't scroll when there's overflow - #170

### PR DESCRIPTION
**Pull request by dougrday**
See https://github.com/Requarks/wiki/issues/170

### CODE REVIEW

**Comment by Bamieh** [comment]
any updates on this? @NGPixel

File: client/scss/components/body.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.body {
```
Comment (NGPixel) [inline client/scss/components/body.scss]: This should be inside `base/base.scss` and the class name should be more relevant (e.g. `.content-column`).

**Comment by dougrday** [comment]
@NGPixel I've made the changes you requested.  Please let me know if there's anything else to do.

**Comment by NGPixel** [comment]
I just tested your PR in both Chrome on Windows + Safari on Mac and it displays 2 scrollbars in the right container, a scrollbar in the left sidebar, with the footer being shown at all times.

**Comment by dougrday** [comment]
@NGPixel Hm... I'm not seeing 2 scrollbars in the right container, only a scrollbar (or 2, depending on the width) in the left sidebar.  The footer is shown at all times.

Are you asking for changes to the implementation?

**Comment by NGPixel** [comment]
The footer should not be shown at all times. Nor should the left sidebar have any extra scrollbar.

**Comment by dougrday** [comment]
I don't have time right now to cleanup the UI.  I'll look at it again in January.

**Comment by Bamieh** [comment]
@dougrday I'll check if i have some time to fork your fork !

**Comment by EvanCarroll** [comment]
Come on. The product doesn't work when you upload enough items to require scrolling. Can this get commited. This is a show stopper for people.

**Comment by NGPixel** [comment]
@EvanCarroll The requested changes have not been submitted yet. You can always apply the changes directly to your own installation if it's critical to you.

**Comment by dougrday** [comment]
I don't have enough time to fix atm.  Feel free to fork and make the changes.

**Pull request closed without merging**